packages/ui-kit/src/hooks/useCounter/useCounter.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import axiosInstance from '../../../../../app/storybook/src/axios'
+import { storybookCodeTemplate, RelativeTimeRange, useStorybookAccessToken } from '../../helpers'
+import { useCounter } from './useCounter'
+
+const args = {
+  numerator: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: process.env.STORYBOOK_METRIC_UNIQUE_NAME_1,
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 30
+    }
+  },
+  denominator: {
+    accessToken: '<PROPEL_ACCESS_TOKEN>',
+    metric: process.env.STORYBOOK_METRIC_UNIQUE_NAME_1,
+    timeRange: {
+      relative: RelativeTimeRange.LastNDays,
+      n: 90
+    }
+  }
+}
+
+// This is a function that returns a React component
+const fractionUnicode = (numerator: string, denominator: string) => (
+  <>
+    <sup>{numerator}</sup> / <sub>{denominator}</sub>
+  </>
+)
+
+const Fraction: React.FC<unknown> = () => {}
+
+const meta: Meta<typeof Fraction> = {
+  title: 'Components/Fraction',
+  tags: ['pattern'],
+  component: Fraction,
+  parameters: {
+    imports: ['useCounter', 'useAccessToken'],
+    isFunction: true,
+    codeTemplate: storybookCodeTemplate,
+    transformBody: (body: string) =>
+      body
+        .replace(`useStorybookAccessToken(axiosInstance)`, 'useAccessToken()')
+        .replace(
+          `{
+      ...args.numerator,
+      accessToken
+    }`,
+          `${JSON.stringify(args.numerator)}`
+        )
+        .replace(
+          `{
+      ...args.denominator,
+      accessToken
+    }`,
+          `${JSON.stringify(args.denominator)}`
+        )
+        .concat(' }')
+  }
+} satisfies Meta<typeof Fraction>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const FractionUnicode: Story = {
+  args,
+  render: (args) =>
+    (function Fraction() {
+      const { accessToken } = useStorybookAccessToken(axiosInstance)
+      // useCounter hoooks with query params timeRange set to last 30 days
+      const { data: numerator } = useCounter({ ...args.numerator, accessToken })
+      // useCounter hoooks with query params timeRange set to last 90 days
+      const { data: denominator } = useCounter({ ...args.denominator, accessToken })
+      return <p>{fractionUnicode(numerator?.counter?.value ?? '', denominator?.counter?.value ?? '')}</p>
```

Comment:
`fractionUnicode` method is not shown in the Story source code
<img width="1016" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/88ba5c74-fe2b-48a2-ab14-cde6614e3924">

Reply:
```suggestion
      return <p style={{ display: 'flex' }}>{fractionUnicode(numerator?.counter?.value ?? '', denominator?.counter?.value ?? '')}</p>
```
to fix the following
<img width="178" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/860c7ef3-758d-4739-a3a1-23dddef349ec">